modules/serverless-creating-subscriptions-kn.adoc

```diff
@@ -17,16 +17,19 @@ To create subscriptions using the `kn` CLI, ensure that:
 
 .Procedure
 
-* Create a subscription to connect a sink to a channel.
+* Create a subscription to connect a sink to a channel:
 +
 [source,terminal]
 ----
 $ kn subscription create <subscription_name> \
-  --channel <Group:Version:Kind>:<channel_name> \
-  --sink <sink_prefix>:<sink_name> \
-  --sink-reply <sink_prefix>:<sink_name> \
-  --sink-dead-letter <sink_prefix>:<sink_name>
-----
+  --channel <Group:Version:Kind>:<channel_name> \ <1>
+  --sink <sink_prefix>:<sink_name> \ <2>
+----
+<1> `--channel` specifies the source for cloud events that should be processed. You must provide the channel name. If you are not using the default channel that is backed by the Channel resource, you must prefix the channel name with the `<Group:Version:Kind>` for the specified channel type. For example, this will be `messaging.knative.dev:v1beta1:KafkaChannel` for a Kafka backed channel.
+<2> `--sink` specifies the target destination to which the event should be delivered. By default, the `<sink_name>` is interpreted as a Knative service of this name, in the same namespace as the subscription. You can specify the type of the sink by using one of the following prefixes:
+`ksvc`:: A Knative service.
+`channel`:: A channel that should be used as destination. Only default channel types can be referenced here.
+`broker`:: An Eventing broker.
```

Comment:
Let's please keep this for another issue / PR - as I stated previously, I don't want to get too sidetracked with work for services docs since that's not really what this section is about, and explaining them here is just a minor section to make folks aware of the options available. We have an issue in the backlog to improve services docs already, but feel free to comment on that re the kn config, or add another issue to add reference docs for that.

Reply:
Thanks, I understand that this PR is already kind of derailed (sorry for that).

So I opened an issue at https://github.com/openshift/openshift-docs/issues/30746 or do you refer to a JIRA ticket ? (and if so, which one ?)

Reply:
moved to JIRA